elasticsearch-log/config.ini

```diff
@@ -1,6 +1,25 @@
-#example for config
-[elasticsearch-log]
-insightFinder_license_key =
-insightFinder_project_name =
-insightFinder_user_name =
-sampling_interval =
+[insightfinder]
+license_key =
+project_name =
+user_name =
+sampling_interval = 1
+if_http_proxy =
+if_https_proxy =
+
+[reporting_parameters]
+serverurl =
+reporting_interval = 1
+
+[elastic_search]
+elasticsearchhost = localhost
+elasticsearchport = 9200
+elasticsearchindex = syslog
+timefieldname = @timestamp
+istimestamp = true
+hostnamefield = host
+dateformatinjodatime = yyyy-MM-dd'T'HH:mm:ss
+dateformatinstrptime = %Y-%m-%dT%H:%M:%S
+datatimezone = Asia/Shanghai
+localtimezone = US/Eastern
+#format for prev_endtime: yyyymmddhhmmss (ex: 20190819155642)
+prev_endtime = 20190819155642
```

Comment:
can you explain  how this is used vs reporting_interval?

Reply:
Reporting interval is referenced when prev_endtime is not available or not known.

Reply:
to keep aligned with our other agents, config.ini should be entirely controlled by the end user and the agent should be stateless. so, just use reporting_interval and drop prev_endtime.